### PR TITLE
inter-region tutorial fix

### PR DESCRIPTION
**Pull request by dPhanekham**
This updates the inter_region_reports tutorial to use the zone flag instead of zones and extra_zones.

This addresses issue #3947 and issue #3746 

It also updates references to Data Studio to instead reference Looker Studio, the new branding.

### CODE REVIEW

File: tutorials/inter_region_reports/README.md [hunk 1]
```diff
@@ -236,17 +236,16 @@ other options for each VM.
...
...west3-a,us-west1-a],[europe-west3-a,us-west2-a],[europe-west4-a,europe-west6-a],[europe-west4-a,northamerica-northeast1-a],[europe-west4-a,southamerica-east1-a],[europe-west4-a,us-central1-a],[europe-west4-a,us-east1-b],[europe-west4-a,us-east4-a],[europe-west4-a,us-west1-a],[europe-west4-a,us-west2-a],[europe-west6-a,northamerica-northeast1-a],[europe-west6-a,southamerica-east1-a],[europe-west6-a,us-central1-a],[europe-west6-a,us-east1-b],[europe-west6-a,us-east4-a],[europe-west6-a,us-west1-a],[europe-west6-a,us-west2-a],[northamerica-northeast1-a,southamerica-east1-a],[northamerica-northeast1-a,us-central1-a],[northamerica-northeast1-a,us-east1-b],[northamerica-northeast1-a,us-east4-a],[northamerica-northeast1-a,us-west1-a],[northamerica-northeast1-a,us-west2-a],[southamerica-east1-a,us-central1-a],[southamerica-east1-a,us-east1-b],[southamerica-east1-a,us-east4-a],[southamerica-east1-a,us-west1-a],[southamerica-east1-a,us-west2-a],[us-central1-a,us-east1-b],[us-central1-a,us-east4-a],[us-central1-a,us-west1-a],[us-central1-a,us-west2-a],[us-east1-b,us-east4-a],[us-east1-b,us-west1-a],[us-east1-b,us-west2-a],[us-east4-a,us-west1-a],[us-east4-a,us-west2-a],[us-west1-a,us-west2-a]]
```
Comment (s-deitz) [inline tutorials/inter_region_reports/README.md:242]: Too bad we're making you do this with the deprecation of those flags..

We used to have a way of creating zip and cross products across flags. Is there some support in YAML that we can use to create this list of lists?

If we can't do that, is there a bash command we can add to make this easier to generate?

File: tutorials/inter_region_reports/README.md [hunk 2]
```diff
@@ -308,18 +307,17 @@ other options for each VM.
...
```
Comment (s-deitz) [inline tutorials/inter_region_reports/README.md:313]: If this is the same list, is there a way to define it in one place?

File: tutorials/inter_region_reports/data/simple_region_iperf.yaml [hunk 3]
```diff
@@ -1,14 +1,10 @@
...
   flag_matrix: inter_region
-  flag_matrix_filters:
-    # For each zone:extra_zone pair where zone != extra_zone.
-    inter_region: "zones < extra_zones"
   flag_matrix_defs:
     inter_region:
       gce_network_tier: [premium]
-      zones: [us-central1-a,us-east1-b]
-      extra_zones: [us-central1-a,us-east1-b]
+      zone: [[us-central1-a,us-east1-b]]
```
Comment (s-deitz) [inline tutorials/inter_region_reports/data/simple_region_iperf.yaml:7]: Well, this looks better. Is there a way to do this without the flag matrix? A list in flags perhaps, or does that not work?

**Comment by bvliu** [comment]
Sorry about the churn, looks like we were working on a fix at the same time. I have a general fix for the tutorials in progress 
here which should allow flag_matrix_defs to work with multi string flags: https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/pull/4038. I'll look to submit this by early next week in which case this change may not be needed. Thanks!

**Comment by dPhanekham** [comment]
No, worries, if there is a better fix on the way, I can close this PR.

**Pull request closed without merging**